src/connectors/query-rules/__tests__/connectQueryRules-test.ts

```diff
@@ -136,7 +140,7 @@ See documentation: https://www.algolia.com/doc/api-reference/widgets/query-rules
           instantSearchInstance,
           state: helper.state,
           results: new SearchResults(helper.state, [
-            { hits: [], userData: [{ banner: 'image.png' }] },
+            createAlgoliaResponse({ userData: [{ banner: 'image.png' }] }),
```

Comment:
TS fails, but that's fixed by https://github.com/DefinitelyTyped/DefinitelyTyped/pull/36859